packages/core/src/tools/mcp-tool.ts

```diff
@@ -32,10 +39,22 @@ export class DiscoveredMCPTool extends BaseTool<ToolParams, ToolResult> {
       name,
       `${serverToolName} (${serverName} MCP Server)`,
       description,
-      parameterSchema,
+      { type: Type.OBJECT }, // this is a dummy Schema for MCP, will be removed when construct the FunctionDeclaration
```

Comment:
We either give it a dummy Schema to make the BaseTool constructor happy or we need to change BaseTool constructor's API to allow us not setting the parameterSchema. I feel this way is less impact on Gemini_cli, but open to other ideas.